feature/default_sets.go

```diff
@@ -0,0 +1,28 @@
+package feature
+
+import "github.com/lightningnetwork/lnd/lnwire"
+
+// defaultSets are the default set descriptors for generating feature vectors.
+// Each set is annotated with the corresponding identifier from BOLT 9
+// indicating where it should be advertised.
+var defaultSets = map[lnwire.FeatureBit]map[Set]struct{}{
```

Comment:
Should this commit be the first, to build on top of it with the selection of features in different contexts?

Reply:
done